common-files/utils/logger.mjs

```diff
@@ -11,7 +11,7 @@ const LOGGER_TIME_STRING = 'yyyy-mm-dd HH:MM:ss.l';
 const getInstance = () => {
   const pinoOptions = {
     level: config.LOG_LEVEL || 'info',
-    messageKey: 'message',
+    base: undefined,
```

Comment:
Wouldn't it be `undefined` by default if we don't set it?

Reply:
Nope, it has some values and are printing the `host` and `pid` which are unnecessary for us.